### DISCUSSION TITLE
Missing features compared to Goku?

### DISCUSSION THREAD

**Discussion by arabshapt** [evan-liu/karabiner.ts#120] [Q&A]
Hi, are there any missing features compared to Goku or pure karabiner config? Does it support everything that karabiner does? What are the cons compared to Goku?

**Comment by evan-liu** [comment]
Hi. It supports everything that Karabiner config JSON supports (for [complex_modifications](https://karabiner-elements.pqrs.org/docs/json/complex-modifications-manipulator-definition/)). I don't know if there are any missing features compared to Goku, though; please let me know if you find any. 

The cons compared to Goku might be fewer resources like tutorials and examples as Goku has been around for much longer.
